### DISCUSSION TITLE
When using SVM as the model, should I always use KernelExplainer?

### DISCUSSION THREAD

**Discussion by Congren-Dai** [shap/shap#2991] [General]
If I use Explainer() instead of KernelExplainer, when algorithm = "auro", the algorithm would be the permutation. So, I'm not sure if the permutation is suitable for SVM.

**Comment by jainraj** [comment]
The KernelExplainer is not related to Kernels in SVMs. 

KernelExplainer builds SHAP model for each sample and would use whatever function/model you provide to calculate model outputs. You could use KernelExplainer for NNs and Permutation for SVMs - doesn't matter syntactically.
